src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5665,25 +5670,22 @@ namespace AirflowNetworkBalanceManager {
 
 							VFObj.LinkageSurfaceData( j ).SurfaceResistanceFactor = StefanBoltzmann / ( DuctSurfResistance + SpaceResistance + ZoneSurfResistance );
 
-							Real64 hrj =  VFObj.LinkageSurfaceData( j ).SurfaceResistanceFactor * ( TDuctSurf_K + TSurfj_K ) * ( pow_2( TDuctSurf_K ) + pow_2( TSurfj_K ) );
+							Real64 hrj = VFObj.LinkageSurfaceData( j ).SurfaceResistanceFactor * ( TDuctSurf_K + TSurfj_K ) * ( pow_2( TDuctSurf_K ) + pow_2( TSurfj_K ) ) / DuctSurfArea;
```

Comment:
So these temperatures in Kelvin really are supposed to be added?  I'm having difficulty grokking it from this diff; I can study the whole algorithm a little more if needed.